### ISSUE TITLE
Toolhead control font size broken on Firefox

### ISSUE THREAD

**Issue by 9R** [mainsail-crew/mainsail#958]
### Mainsail Version:

v2.2.1

### Browser:

Firefox

### Device:

Laptop

### Operating System:

Linux

### What happened?

When using Firefox the font on the toolhead control style 'circle' is not rendered as expected.

Firefox screenshot:

![Screenshot from 2022-07-10 16-03-49](https://user-images.githubusercontent.com/381298/178148627-e0afb85d-df8f-4bb3-9981-15754b52e2c5.png)

### What did you expect to happen instead?

Font with correct size when using Chromium
![Screenshot from 2022-07-10 16-04-22](https://user-images.githubusercontent.com/381298/178148633-19229507-67d2-4d06-ac5d-694604682857.png)
.

### How to reproduce this bug?

Open the UI with firefox and activate toolhead control with style circle

### Additional information:

Firefox version: 101.0.1

**Comment by th33xitus** [comment]
> Open the UI with firefox and activate toolhead control with style circle

Unable to reproduce. Although im using Firefox on Windows.

![image](https://user-images.githubusercontent.com/31533186/178148891-31500d14-5601-4b45-88ba-3962853fa21a.png)

**Comment by 9R** [comment]
I have done some further testing.
The issue arises on a laptop with high resolution display (12.5" with 3000x2000). I have tested other devices with fullHD and 4K displays, that also run Linux and latest Firefox. There the font is rendered correctly. Strangely enough when I compare eck the font parameters with dev tools from FIrefox they are the same on the offending device and the one with the 4K display that renders toolhead control correctly.
I also tried using dev tools to modify font-size manually which was not possible, Below a certain threshold the font will not get any smaller, but I am no expert in webdesign so I'm not sure if i have exhausted all possibilities.
Non of the tested computers use any text-scaling or accessibility features that I know of. I have not noticed any similar rendering errors elsewhere in mainsail or on other websites with the laptop that exhibits the behavior.

I'm unsure how to proceed, since this seems to be isolated to a single device for now and I do not know how to reproduce the behavior on other machines and impact is mostly visual and there are probably more important issues to resolve.
